ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricRegistryOwner.scala

```diff
@@ -25,7 +25,7 @@ class MetricRegistryOwner(
       registry <- ResourceOwner.forValue(() => new MetricRegistry).acquire()
       _ <- acquireSlfjReporter(registry)
       metricsReporter <- acquireMetricsReporter(registry)
-      _ = metricsReporter.start(reportingInterval.toSeconds, TimeUnit.SECONDS)
+      _ = metricsReporter.start(reportingInterval.toMillis, TimeUnit.MILLISECONDS)
```

Comment:
This is an extra bug fix for errors when user defines log interval smaller than a second.